deanonymization/check_bitcoin_addresses.go

```diff
@@ -97,6 +142,9 @@ func ExtractBitcoinAddress(osreport *report.OnionScanReport, anonreport *report.
 				if ValidA58([]byte(result)) {
 					anonreport.BitcoinAddresses = append(anonreport.BitcoinAddresses, result)
 					osc.Database.InsertRelationship(osreport.HiddenService, "snapshot", "bitcoin-address", result)
+				} else if ValidP58([]byte(result)) {
+					anonreport.BitcoinAddresses = append(anonreport.BitcoinAddresses, result)
```

Comment:
We should define a new field in AnonymityReport for BitcoinPrivateKeys, these don't belong in BitcoinAddresses